server.js

```diff
@@ -16,6 +16,7 @@ const DEPLOY_URL = process.env.NODE_ENV === "production" ? "" : "http://localhos
 const PORT = process.env.HTTP_PORT || LOCAL_PORT;
 const TwitchStrategy = passportTwitch.Strategy;
 const app = express();
+var token = ""
```

Comment:
I tried a few different ways but ultimately this was the easiest and most straightforward